src/actions/swift/colorset.ts

```diff
@@ -54,6 +53,14 @@ export default {
 
       const colorset = getOrCreateColorset(`${colorsetPath}/Contents.json`);
 
+      /**
+       * We need to resolve the references for asymmetric tokens otherwise there isn't
+       * and RGB color value to .
```

Comment:
How was this meant to read?

Reply:
lol, can you tell who got distracted mid-way through a sentence and forgot to come back to it 🙈

Thanks :D